pkg/kev/config/extensions_test.go

```diff
@@ -0,0 +1,117 @@
+package config_test
+
+import (
+	"bytes"
+
+	"github.com/appvia/kev/pkg/kev/config"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/yaml.v3"
+)
+
+var _ = Describe("Extentions", func() {
+
+	Describe("parsing", func() {
+		var (
+			parsedCfg  config.K8SServiceConfig
+			k8s        config.K8SServiceConfig
+			extensions = make(map[string]interface{})
+			err        error
+		)
+
+		BeforeEach(func() {
+			k8s = config.K8SServiceConfig{}
+		})
+
+		JustBeforeEach(func() {
+			var buf bytes.Buffer
+			err := yaml.NewEncoder(&buf).Encode(map[string]interface{}{
+				"x-k8s": k8s,
+			})
+			Expect(err).To(Succeed())
+
+			err = yaml.NewDecoder(&buf).Decode(&extensions)
+			Expect(err).To(Succeed())
+		})
+
+		Context("works with defaults", func() {
+			BeforeEach(func() {
+				k8s.Workload.Replicas = 10
+				k8s.Workload.LivenessProbe.Type = config.ProbeTypeNone.String()
+			})
+
+			It("Works with defaults", func() {
```

Comment:
would be great to differentiate or put some more detail in that case description. Currently it's identical to the context description...